pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -499,6 +503,8 @@ func (l *LibvirtDomainManager) SetGuestTime(vmi *v1.VirtualMachineInstance) erro
 	return nil
 }
 
+var GetLoopbackAddress = ip.GetLoopbackAddress
```

Comment:
I think it doesn't have to be exportable, please change to it `getLoopbackAddress`
Also, what's the difference between this and `loopbackAddress`?
Perhaps you can simply get the address once?

Reply:
done, thanks